docs/cli-reference.md

```diff
@@ -7,12 +7,12 @@
   git clone https://github.com/openshift/nodejs-ex && cd nodejs-ex
   odo create nodejs
   odo push
-  
+
```

Comment:
cli-reference.md is automatically generated. Can you revert your changes to `cli-reference.md` ?